swarm/network/types.go

```diff
@@ -0,0 +1,70 @@
+package network
```

Comment:
Why to move this code to file named types.go? This utility names should be avoided. I propose name network.go instead types.go.

Reply:
sure